treeherder/model/management/commands/cycle_data.py

```diff
@@ -62,6 +63,11 @@ def handle(self, *args, **options):
                                                 options['sleep_time'])
             self.debug("Deleted {} jobs from {}".format(rs_deleted,
                                                         repository.name))
+            if repository.expire_performance_data:
```

Comment:
Yep thanks I saw that for the model, but I was trying to figure out the 'expire_performance_data' value in /fixtures/repository.json overrides that default True value set in the model, so that expiring data can be disabled? I don't see a check for that setting in the .json anywhere... but I just read the django documentation about fixtures, looks like you run a .manage.py loaddate <fixturename> command...